### PR TITLE
docs(illustrations): add popover to spot illustrations

### PR DESCRIPTION
**Pull request by ninamarina**
## Description
Jira: https://dialpad.atlassian.net/browse/DLT-1395

Adds a popover for the Illustrations in the design page. 
The popover contains:

- The name and description
- Keywords: for now we are not adding them because there is no search, and also not that many illustrations
- SVG code: the raw SVG code with hex colors so that it can be used outside the application (like in presentations)
- Vue component with copy button
- Figma link

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [x] Update, remove, or extend all affected documentation.
 - [x] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

## Obligatory GIF (super important!)

illus-popover

### CODE REVIEW

File: apps/dialtone-documentation/docs/.vuepress/baseComponents/BaseIcon.vue [hunk 1]
```diff
@@ -7,132 +7,149 @@
...
-    },
+const selected = ref(false);
+const iconComponent = ref(null);
 
-    cardIconClass () {
-      return 'dialtone-icon-card__icon--autosize';
-    },
-  },
+const selectedStatus = computed(() => selected.value ? 'yes' : 'no');
+const cardIconClass = computed(() => 'dialtone-icon-card__icon--autosize');
```
Comment (braddialpad) [inline apps/dialtone-documentation/docs/.vuepress/baseComponents/BaseIcon.vue]: I know you just copied what was already there, but this seems pointless. We can just apply `dialtone-icon-card__icon--autosize` class directly to the element.

File: apps/dialtone-documentation/docs/.vuepress/theme/assets/less/dialtone-docs.less [hunk 2]
```diff
@@ -436,6 +436,13 @@ a.header-anchor {
   display: flex;
   width: 100%;
   height: 100%;
+
+  .d-popover {
+    height: 100%;
+    > div {
+      height: 100%;
+    }
+  }
```
Comment (braddialpad) [inline apps/dialtone-documentation/docs/.vuepress/theme/assets/less/dialtone-docs.less:445]: hmm do we need to override th d-popover class here? can we just apply this on the instance(s) where it's needed using class props and utility classes?

**Review by yorbi-dp — APPROVED** [review]
Thank you Nina!
L-GREAT-TM! Nice work.

File: apps/dialtone-documentation/docs/.vuepress/client.js [hunk 3]
```diff
@@ -18,6 +18,18 @@ import ComponentCombinator from './baseComponents/ComponentCombinator.vue';
...
+import SpotPublish from '@dialpad/dialtone/lib/dist/vue/spot/SpotPublish.vue';
+import SpotWirelessScreenshare from '@dialpad/dialtone/lib/dist/vue/spot/SpotWirelessScreenshare.vue';
+import SpotFemaleLaptopTyping from '@dialpad/dialtone/lib/dist/vue/spot/SpotFemaleLaptopTyping.vue';
+import SpotMaleLaptopTyping from '@dialpad/dialtone/lib/dist/vue/spot/SpotMaleLaptopTyping.vue';
+import SpotFileUpload from '@dialpad/dialtone/lib/dist/vue/spot/SpotFileUpload.vue';
+import SpotBrowserTableGraph from '@dialpad/dialtone/lib/dist/vue/spot/SpotBrowserTableGraph.vue';
+import SpotBrowserListCallout from '@dialpad/dialtone/lib/dist/vue/spot/SpotBrowserListCallout.vue';
+import SpotMind from '@dialpad/dialtone/lib/dist/vue/spot/SpotMind.vue';
+import SpotBlankSpace from '@dialpad/dialtone/lib/dist/vue/spot/SpotBlankSpace.vue';
+import SpotVectorVortex from '@dialpad/dialtone/lib/dist/vue/spot/SpotVectorVortex.vue';
```
Comment (braddialpad) [inline apps/dialtone-documentation/docs/.vuepress/client.js]: I'll see if I can figure out the problem with that svgloader component. Would be nice if we didn't have to do this.

**Review by braddialpad — COMMENTED** [review]
New popover looks great! so much better than the old one. Couple small things.

File: apps/dialtone-documentation/docs/.vuepress/theme/assets/less/dialtone-docs.less (hunk 2 above)
Comment (ninamarina) [reply apps/dialtone-documentation/docs/.vuepress/theme/assets/less/dialtone-docs.less:445]: I tried but couldn't, there is no class prop for the anchor and also passing a utility class to dt-popover doesn't apply the class

File: apps/dialtone-documentation/docs/.vuepress/client.js (hunk 3 above)
Comment (ninamarina) [reply apps/dialtone-documentation/docs/.vuepress/client.js]: I suspect it has something to do with the defs but couldn't figure it out. Inspecting the svg, everything seems ok.

**Comment by braddialpad** [comment]
@ninamarina Just pushed a fix for the svg color issue. `vite-svg-loader` had svgo enabled by default (svg minifier/optimizer) and this was not playing nice with our css variables. I just disabled it and it has fixed the issue.

**Comment by braddialpad** [comment]
You'll still need to revert back to the prior method in this PR, but I'll pre-approve for when you do so.

**Review by braddialpad — APPROVED** [review]

**Pull request merged**